src/backend/app/Services/AgentSoldApplianceService.php

```diff
@@ -135,15 +135,15 @@ public function processSaleFromRequest(AgentSoldAppliance $agentSoldAppliance, a
         // create agent transaction
         $agentTransactionData = [
             'agent_id' => $agent->id,
-            'device_id' => $agent->device_id,
+            'device_id' => $agent->mobile_device_id,
             'status' => 1,
         ];
```

Comment:
Could we also rename this column in `agent_transaction` ? It would be great to have it consistent.